src/web/Html.tsx

```diff
@@ -26,16 +26,19 @@ function isObjectBehindCamera(el: Object3D, camera: Camera) {
   return deltaCamObj.angleTo(camDir) > Math.PI / 2
 }
 
-function objectScale(el: Object3D, camera: Camera) {
-  if (camera instanceof PerspectiveCamera) {
-    const objectPos = v1.setFromMatrixPosition(el.matrixWorld)
-    const cameraPos = v2.setFromMatrixPosition(camera.matrixWorld)
-    const vFOV = (camera.fov * Math.PI) / 180
-    const dist = objectPos.distanceTo(cameraPos)
+function objectScale(el, camera) {
+  const objectPos = v1.setFromMatrixPosition(el.matrixWorld)
+  const cameraPos = v2.setFromMatrixPosition(camera.matrixWorld)
+  const vFOV = (camera.fov * Math.PI) / 180
+  const dist = objectPos.distanceTo(cameraPos)
+
+  if (camera instanceof OrthographicCamera) {
+    return 1 / ((2 * Math.tan(vFOV / 2) * dist) / camera.zoom)
```

Comment:
do you think we could extract this into `const objScale = 1 / ((2 * Math.tan(vFOV / 2) * dist)` to avoid the dupe & make it clearer what the difference is between perspective camera  & orthographic?

Reply:
Ahhh of course, without a doubt! I'll fix it up now.